MetaMask/metamask-extension
# move search icon || update imports || add to story

## Explanation

Currently, 
SearchIcon is stored in ui/components/ui/search-icon/search-icon.component.js  


Should be stored in ui/components/ui/icon/search-icon.component.js and added to  icon stories 


## More Information
* Fixes #13563

## Screenshots/Screencaps



### Before


![image](https://user-images.githubusercontent.com/23730603/173570479-330db860-9d7e-4a64-ba2c-3fbc073cf3e1.png)

### After


![image](https://user-images.githubusercontent.com/23730603/173570398-a955a30b-ce34-4c74-88d9-cb1db989428c.png)

## Manual Testing Steps

<!--
How should reviewers and QA manually test your changes? For instance:

- Start storybook
- Go to /story/ui-components-ui-icon-icon-stories-js--default-story

## Pre-Merge Checklist


### + If there are functional changes:
